docs/deployment/feature_gates.md

```diff
@@ -208,3 +209,4 @@ A *General Availability* (GA) feature is also referred to as a *stable* feature.
 | ShootManagedIssuer              | `gardenlet`                       | Enables the shoot managed issuer functionality described in GEP 24.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
 | VPAForETCD                      | `gardenlet`, `gardener-operator`  | Enables VPA for `etcd-main` and `etcd-events`, regardless of HVPA enablement.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
 | VPAAndHPAForAPIServer           | `gardenlet`, `gardener-operator`  | Enables an autoscaling mechanism for `kube-apiserver` of shoot or virtual garden clusters, and the `gardener-apiserver`. They are scaled simultaneously by VPA and HPA on the same metric (CPU and memory usage). The pod-trashing cycle between VPA and HPA scaling on the same metric is avoided by configuring the HPA to scale on average usage (not on average utilization) and by picking the target average utilization values in sync with VPA's allowed maximums. The feature gate takes precedence over the `HVPA` feature gate when they are both enabled. |
+| NewWorkerPoolHash               | `gardenlet`                       | Enables usage of the new worker pool hash calculation. The new calculation supports rolling worker pools if `kubeReserved`, `evicitonHard` or `cpuManagerPolicy` in the `kubelet` configuration are changed. All provider extensions must be upgraded to support this feature first.                                                                                                                                                                                                                                                                                  |
```

Comment:
```suggestion
| NewWorkerPoolHash               | `gardenlet`                       | Enables usage of the new worker pool hash calculation. The new calculation supports rolling worker pools if `kubeReserved`, `evicitonHard` or `cpuManagerPolicy` in the `kubelet` configuration are changed. All provider extensions must be upgraded to support this feature first. Existing worker pools are not immediately migrated to the new hash variant, since this would trigger a replacements of all nodes. The migration happens, when a rolling update was triggered according to the old hash version calculation.                                                                                                                                                                                                                                                                                  |
```

Might be an important information for operator. They can safely switch on the feature gate.

Reply:
Good idea, thanks. I've slightly tweaked the wording of the last sentence. If `NewWorkerPoolHash` is enabled, then after the shoot has been reconciled once, then a node pool roll can also be triggered by the new fields.